packages/frameworks-nuxt/src/module.ts

```diff
@@ -0,0 +1,294 @@
+import {
+  defineNuxtModule,
+  addImports,
+  addPlugin,
+  addTypeTemplate,
+  createResolver,
+  addRouteMiddleware,
+  useLogger,
+} from "@nuxt/kit"
+import { defu } from "defu"
+
+const configKey = "authJs" as const
```

Comment:
will there be cases where we need to customize this value.

Reply:
Not really. Would you like to change it to a simple const?

Reply:
I modified to use simple 'auth' as configKey